src/App.jsx

```diff
@@ -1,37 +1,43 @@
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
 import reactLogo from './assets/react.svg'
 import viteLogo from '/vite.svg'
 import React from 'react'
 import './App.css'
 import TodoList from './TodoList'
 import AddTodoForm from './AddTodoForm'
+import InputWithLabel from './InputWithLabel'
 
 
+function useSemiPersistentState() {
+  const [todoList, setTodoList] = React.useState(JSON.parse(localStorage.getItem("savedTodoList")));
```

Comment:
Pro tip: You can remove the `React.` since you already import `useState` on line 1. Afterwards you can also remove the `import React from 'react'` statement on line 4

Reply:
React. , import React from 'react' - were removed